blueprint/src/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "base-component",
+  "description": "oc-template-react base component",
+  "version": "0.0.1",
```

Comment:
npm correctly threat every version that is below 1.0.0, but we could go with 1.0.0 agree